Makefile

```diff
@@ -8,5 +10,30 @@ fmt:
 vet:
 	go vet ./...
 
+build-image:
+	${CONTAINER_RUNTIME} build -t ${IMG} .
+
 cmd: fmt vet
 	go build -ldflags="-w -s" -o bin/addon github.com/konveyor/tackle2-addon-windup/cmd
+
+.PHONY: test-e2e
+START_MINIKUBE = ./bin/start-minikube.sh
+INSTALL_TACKLE = ./bin/install-tackle.sh
+start-minikube:
+	mkdir -p $(dir $(START_MINIKUBE)) ;\
+	curl -sSLo $(START_MINIKUBE) https://raw.githubusercontent.com/konveyor/tackle2-operator/main/hack/start-minikube.sh ;\
+	chmod +x $(START_MINIKUBE) ;\
+	$(START_MINIKUBE);
+
+install-tackle:
```

Comment:
Same recommendation here as for start-minikube, we don't want to "curl" this every time. Just if it isn't available.